actionpack/lib/action_controller/metal/helpers.rb

```diff
@@ -53,9 +53,8 @@ class << self; attr_accessor :helpers_path; end
     include AbstractController::Helpers
 
     included do
-      class_attribute :helpers_path, :include_all_helpers
-      self.helpers_path ||= []
```

Comment:
I hesitated at that, but ultimately convinced myself the `||=` is a lie: `class_attribute` will always define a method on the current class, with the value set to nil... so AFAICS, this is no different from if it were a straight assignment.